<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RtmpPushClass</class>
 <widget class="QMainWindow" name="RtmpPushClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RtmpPush</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="6,4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>画面源：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="vDevices_box"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>音频源：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="aDevices_box"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>码率：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="Bitrates"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>帧率：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="Framerate"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>分辨率：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="resolution_box"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>流媒体地址：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rtmp_edit"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="8,2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,6,2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(56, 56, 56);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="vframe_lable">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(100, 100, 100);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(56, 56, 56);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionstart"/>
    <addaction name="actionend"/>
    <addaction name="separator"/>
    <addaction name="actionsetting"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuVDevices">
    <property name="title">
     <string>VDevices</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuADevices">
    <property name="title">
     <string>ADevices</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuVDevices"/>
   <addaction name="menuADevices"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionstart">
   <property name="text">
    <string>start</string>
   </property>
  </action>
  <action name="actionend">
   <property name="text">
    <string>end</string>
   </property>
  </action>
  <action name="actionsetting">
   <property name="text">
    <string>setting</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MediaPushWindow.qrc"/>
 </resources>
 <connections/>
</ui>
